src/test/java/org/simplify4u/plugins/ValidationChecksumTest.java

```diff
@@ -57,7 +58,7 @@ public void testValidationChecksumBuilderNullFile() {
     @Test
     public void testValidationChecksumBuilderArtifactsNull() {
         final Builder builder = new ValidationChecksum.Builder().destination(checksumdirectory);
-        assertThatCode(() -> builder.artifacts(null).build())
+        assertThatThrownBy(() -> builder.artifacts(null))
```

Comment:
Which method throw exception?
why change assertThatCode -> assertThatThrownBy?

Reply:
I see, here we can have simply:
```
        assertThatCode(() -> builder.artifacts(null))
                .isExactlyInstanceOf(NullPointerException.class);
```

in next test `testValidationChecksumBuilderArtifactsNotProvided` we check build method

Reply:
assertThatThrownBy is more correct, it will fail if nothing has been thrown?

Reply:
I've changed it to assertThatCode to be consistent with the other tests.